src/pages/app/menus/AddonMenuItem.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+  import MenuItem from "@/common/MenuItem";
+  import Tooltip from "@/common/Tooltip";
+  import { layout, dispatchAddonSelected,openDispatchAddon } from "@/manager/layout";
+  import {
+    selectedDocsInProject,
+  //   addSelectedDocsToProject,
+  //   removeSelectedDocsFromProject,
+  } from "@/manager/addons";
+  import { handlePlural } from "@/util/string";
+
+  export let addon;
+
+  $: scope = "none"
```

Comment:
ProjectMenuItem.svelte has: `$: scope =
    $layout.selected.length > 0 ? selectedDocsInProject(project) : "none";`
We might need it back.

Reply:
I believe that code is specific to adding and removing documents from a project and is not needed in this context.